modules/local/filter_stats.nf

```diff
@@ -10,7 +10,7 @@ process FILTER_STATS {
         mode: params.publish_dir_mode,
         saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:'') }
 
-    conda (params.enable_conda ? "pandas=1.1.5--py39ha9443f7_0" : null)
+    conda (params.enable_conda ? "pandas=1.1.5=py39ha9443f7_0" : null)
```

Comment:
Would you be able to test if the line:
`conda (params.enable_conda ? "pandas=1.1.5" : null)`
works as well, which would follow the guidelines that the build should not be pinned.

Reply:
Ahh, didn't know about that. Your suggestion above seems to work!